static/sass/elements/chromedash-metrics.scss

```diff
@@ -36,13 +36,20 @@
       }
     }
 
+    > :first-child {
+      @include flex(1);
+    }
+
+    > :nth-child(2) {
+      @include flex(2);
```

Comment:
There are two more children of stack-rank-list (one was for loading text and the other one I couldn't understand) which were hidden and didn't have sub tags either when everything is loaded. I wonder if this will have any bad effect on those if they can change during the loading or something like that. But I guess that wouldn't matter as much. I assume they will never have children so it should be good.